src/GameState/Board.tsx

```diff
@@ -1,32 +1,34 @@
-import {
-  Stack,
-  HStack,
-  Box,
-} from "native-base";
-import React from "react";
-import { Marble } from "./Marble";
+import { Stack, HStack, Box } from 'native-base';
+import React from 'react';
+import { Marble } from './Marble';
 
 export const Board = (props: any) => {
   const board = props.board;
+  const boardSize = props.preview ? 4 : 12;
+  const marbleColorSize = props.preview ? 4 : 12;
+  const marbleEmptySize = props.preview ? 2 : 4;
   return (
     <Stack mb={4}>
       {board.map((row: [], rowIndex: number) => (
         <HStack alignItems="center" key={rowIndex}>
           {row.map((cell, cellIndex) => (
             <Box
-              size={12}
+              size={boardSize}
               alignItems="center"
               justifyContent="center"
               key={cellIndex}
               bg="#948e8b"
               m={0}
               p={0}
             >
-              <Marble value={cell}/>
+              <Marble
+                value={cell}
+                size={cell === 0 ? marbleEmptySize : marbleColorSize}
```

Comment:
IMO this logic should be handled on the `Marble` component side

Reply:
Do you think? 
Shouldn't the marble component just display a marble without caring about its context?

Reply:
It's not wrong. I don't really like the ... *mamaille* on lines 7 to 9. But okay.